### DISCUSSION TITLE
Loading assets using @script shows erros in JS console

### DISCUSSION THREAD

**Discussion by Jamiul-Bari** [livewire/livewire#8693] [7. Bugs (archived)] [answered]
### Livewire version

v3.5.4

### Laravel version

v11.17.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. clone the [reposity](https://github.com/Jamiul-Bari/livewire-script-loading-asset-bug.git)
2. run `composer install` & `npm install` & `npm run dev`
3. setup .env
4. go to `/` route
5. check the browser console
6. there are comments regarding when the error is shown and when it is not

### Repository or Wirebox URL

https://github.com/Jamiul-Bari/livewire-script-loading-asset-bug.git

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by danie-ramdhani** [comment, marked as answer]
It's not bug, please read the documentation first. Use `@script` to load javascript and use `@assets` to load assets like javascript file and  css file. That spatie livewire filepond directive is loaded assets. See this 
link https://github.com/spatie/livewire-filepond/blob/main/src/LivewireFilepondServiceProvider.php
